### DISCUSSION TITLE
image genaration with stable defusion or flux1

### DISCUSSION THREAD

**Discussion by olumolu** [Jeffser/Alpaca#411] [Ideas]
https://github.com/LykosAI/StabilityMatrix
https://github.com/huggingface/diffusers
Or other services kike defusers it is possible to have image genaration integrated in the alpaca app..
Which can be a great feature to have.

**Comment by Jeffser** [comment]
Sorry but I'm against image generation

**Reply by olumolu** [reply]
True this image generation does not do anything important or useful that much but i find this to be fun... except having fun i could not find anything useful with image generation. I respect your idea.
But for fun i created my dp running it on my system locally...
